flow@ideXcode.md

```diff
@@ -4,15 +4,27 @@ If you have already installed Copilot for Xcode, you must uninstall it before in
 
 ## Step 2: Initial Settings
 
-1 - Go to your computer settings and click on **Security & Privacy**. Give Flow@IDE permissions for the Accessibility API.
+1 - To perform the initial settings, on the application's main screen there are two buttons that will take you to the macOS settings. The first one will take you to the accessibility setting.
+
+<img alt="Accessibility API" src="images/AccessibilityButton.png" width="500px">
+
+Then you must add Flow@IDEExtensionService and enable it.
```

Comment:
it's not necessary to add... the plugin adds it automatically. The user just need to enable it.